ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-stable.yaml

```diff
@@ -107,13 +107,14 @@ tests:
     test:
     - chain: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private-network-mtu-edge-zone
-- as: aws-ipi-disc-priv-sno-f14
+- as: aws-ipi-disc-priv-sno-tp-f14
```

Comment:
If we directly enable TP on the existing ci jobs, we would missing non-TP sno ci.

Reply:
there're still several non-TP sno jobs, such as for 4.15:
```
[root@preserve-gpei-test-fedora openshift-tests-private]# grep "as.*sno" openshift-openshift-tests-private-release-4.15*  |grep -v tp
openshift-openshift-tests-private-release-4.15__amd64-nightly-4.15-upgrade-from-stable-4.14.yaml:- as: aws-ipi-disc-priv-sno-f28
openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml:- as: baremetal-pxe-sno-agent-ipv4-static-connected-f14
openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml:- as: baremetal-sno-agent-ipv4-static-connected-fips-f7
openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml:- as: vsphere-agent-disc-sno-f14
openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml:- as: vsphere-agent-sno-f14
openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml:- as: vsphere-agent-dualstack-sno-f14
openshift-openshift-tests-private-release-4.15__amd64-stable.yaml:- as: baremetal-pxe-sno-agent-ipv4-static-connected-f14
openshift-openshift-tests-private-release-4.15__arm64-nightly.yaml:- as: aws-ipi-sno-lvms-f2
openshift-openshift-tests-private-release-4.15__arm64-nightly.yaml:- as: baremetal-pxe-sno-agent-ipv4-static-connected-f14
openshift-openshift-tests-private-release-4.15__arm64-nightly.yaml:- as: baremetal-sno-agent-ipv4-static-connected-f7
openshift-openshift-tests-private-release-4.15__arm64-stable.yaml:- as: aws-ipi-disc-priv-sno-f14
openshift-openshift-tests-private-release-4.15__arm64-stable.yaml:- as: baremetal-pxe-sno-agent-ipv4-static-connected-f14
openshift-openshift-tests-private-release-4.15__multi-nightly.yaml:- as: aws-ipi-sno-etcd-encryption-arm-f2
openshift-openshift-tests-private-release-4.15__multi-nightly.yaml:- as: aws-ipi-sno-etcd-encryption-amd-f9-destructive

[root@preserve-gpei-test-fedora openshift-tests-private]# grep "as.*sno" openshift-openshift-tests-private-release-4.15*  |grep tp
openshift-openshift-tests-private-release-4.15__amd64-stable.yaml:- as: aws-ipi-disc-priv-sno-tp-f14
openshift-openshift-tests-private-release-4.15__multi-nightly.yaml:- as: azure-ipi-sno-etcd-encryption-tp-amd-f9-destructive
openshift-openshift-tests-private-release-4.15__multi-nightly.yaml:- as: azure-ipi-sno-etcd-encryption-tp-arm-f2
openshift-openshift-tests-private-release-4.15__multi-nightly.yaml:- as: gcp-ipi-sno-etcd-encryption-tp-amd-f28-destructive
openshift-openshift-tests-private-release-4.15__multi-nightly.yaml:- as: gcp-ipi-sno-etcd-encryption-tp-arm-f14
```

Reply:
For `aws-ipi-disc-priv-sno-tp-f14`, it is in stable ci, sounds tp feature does not have priority in stable ci.
For `azure-ipi-sno-etcd-encryption-tp` and `gcp-ipi-sno-etcd-encryption-tp`, once that, there would be no sno non-tp ci jobs in regular nightly ci, that will make sno tp jobs have higher priority than sno non-tp jobs, but actually I think sno non-tp jobs should have higher priority.

I wonder is there some more background to enabled tp on sno jobs? Did we receive some customer case or some bug?

Reply:
> I wonder is there some more background to enabled tp on sno jobs? Did we receive some customer case or some bug?

yes, it's coming from https://issues.redhat.com/browse/OCPBUGS-31040 , the reporter installed SNO with TP featureset using a stable 4.15 build, so I choose this one for covering it in stable build CI testing. 
As to the `azure-ipi-sno-etcd-encryption-tp` and `gcp-ipi-sno-etcd-encryption-tp` they're the only existing sno jobs for azure/gcp, but I don't want to add more sno jobs so just enable TP to them. 

> I think sno non-tp jobs should have higher priority

If we want to keep the normal sno testing in the first place for the platforms, I think we can choose an AWS "f2" sno job to enable TP to make sure this is frequently covered in our CI? as I see there're serveral sno jobs for AWS
```
openshift-openshift-tests-private-release-4.15__amd64-stable.yaml:- as: aws-ipi-disc-priv-sno-f14
openshift-openshift-tests-private-release-4.15__arm64-nightly.yaml:- as: aws-ipi-sno-lvms-f2
openshift-openshift-tests-private-release-4.15__arm64-stable.yaml:- as: aws-ipi-disc-priv-sno-f14
openshift-openshift-tests-private-release-4.15__multi-nightly.yaml:- as: aws-ipi-sno-etcd-encryption-arm-f2
openshift-openshift-tests-private-release-4.15__multi-nightly.yaml:- as: aws-ipi-sno-etcd-encryption-amd-f9-destructive
```

Reply:
> the reporter installed SNO with TP featureset using a stable 4.15 build, so I choose this one for covering it in stable build CI testing.

Our stable ci testing is just for the final sanity testing before release, we are expecting to find out bugs in advance, especially in nightly ci, because all the stable build are prompted from nightly.

To cover the issues in our daily ci, I think we have to add some more sno jobs to cover the case, maybe we can add the jobs in openshift-verification-tests-master__installation-nightly-4.N.yaml ? So that not bring some side effect on the regular ci matrix. From my understanding, tp jobs always have lower priority than non-tp jobs.